ga4gh/cli.py

```diff
@@ -638,7 +641,10 @@ def addClientGlobalOptions(parser):
     parser.add_argument(
         "--workarounds", "-w", default=None, help="The workarounds to use")
     parser.add_argument(
-        "--key", "-k", help="The auth key to use")
+        "--key", "-k",
+        help=(
+         "The auth key to use. Browse to https://<server>/_login to get one."),
+        required=True)
```

Comment:
We shouldn't force the use of auth. This is an entirely optional part of the protocol, which will not be appropriate for a lot of use cases (for example, data like 1000 Genomes is completely open and people will want to run public servers to make their data available without restrictions). So, the default behaviour of the server should remain as it is: serve data with no auth or encryption.

From a CLI perspective, I think mandatory 'options' are very iffy. If an argument really is mandatory, it should be a positional argument IMO.